### PR TITLE
Add warning component for funding market and pool liquidity

### PR DESCRIPTION
**Pull request by pimato**
Pool Liquidity:
Screenshot 2020-06-15 at 21 36 46

Fund Market:
Screenshot 2020-06-15 at 21 37 03

### CODE REVIEW

File: app/src/components/market/common/warning_message/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  color: ${props => props.theme.colors.primaryLight};
+`
+
+type Props = HTMLAttributes<HTMLDivElement>
+
+export const WarningMessage: React.FC<Props> = (props: Props) => {
+  const { ...restProps } = props
+
+  return (
+    <Wrapper {...restProps}>
```
Comment (gabitoesmiapodo) [inline app/src/components/market/common/warning_message/index.tsx:40]:
```suggestion
    <Wrapper {...props}>
```

File: app/src/components/market/common/warning_message/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+`
+
+const Hyperlink = styled.a`
+  color: ${props => props.theme.colors.primaryLight};
+`
+
+type Props = HTMLAttributes<HTMLDivElement>
+
+export const WarningMessage: React.FC<Props> = (props: Props) => {
+  const { ...restProps } = props
```
Comment (gabitoesmiapodo) [inline app/src/components/market/common/warning_message/index.tsx:37]:
```suggestion
```

File: app/src/components/market/common/warning_message/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  return (
+    <Wrapper {...restProps}>
+      <AlertWrapper>
+        <Alert />
+      </AlertWrapper>
+      <Description>
+        Providing liquidity is risky and could result in near total loss. It is important to withdraw liquidity before
+        the event occurs and to be aware the market could move abruptly at any time.{' '}
+        <Hyperlink href="./faq.pdf" target="_blank">
+          More Infos
```
Comment (fvictorio) [inline app/src/components/market/common/warning_message/index.tsx:48]:
```suggestion
          More Info
```

File: app/src/components/market/common/warning_message/index.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  return (
+    <Wrapper {...restProps}>
+      <AlertWrapper>
+        <Alert />
+      </AlertWrapper>
+      <Description>
+        Providing liquidity is risky and could result in near total loss. It is important to withdraw liquidity before
+        the event occurs and to be aware the market could move abruptly at any time.{' '}
+        <Hyperlink href="./faq.pdf" target="_blank">
```
Comment (gabitoesmiapodo) [inline app/src/components/market/common/warning_message/index.tsx:47]: Maybe it would be good to have PDFs an other docs in a separate folder.

**Pull request merged**